res/controllers/Numark-Mixtrack-3.midi.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<!-- Numark Mixtrack 3 and Pro 3 MIDI preset v1.2 -->
+<!-- Numark Mixtrack 3 and Pro 3 MIDI preset v1.3 -->
 <MixxxControllerPreset mixxxVersion="1.12.0+" schemaVersion="1">
     <info>
 	<name>Numark Mixtrack Pro 3</name>
```

Comment:
Change this to `Numark Mixtrack (Pro) 3` so users with the non-Pro version know to use this mapping.